releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty.yml

```diff
@@ -224,15 +224,22 @@ services:
       - data
 
   rulesengine:
-    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine:master
+    image: emqx/kuiper:0.3.0
     ports:
-      - "48075:48075"
-    container_name: edgex-support-rulesengine
-    hostname: edgex-support-rulesengine
+      - "9081:9081"
+      - "20498:20498"
+    container_name: edgex-kuiper
+    hostname: edgex-kuiper
     networks:
       edgex-network:
         aliases:
-          - edgex-support-rulesengine
+          - edgex-kuiper
+    environment:
+      EDGEX_SERVER: edgex-core-data
```

Comment:
Do you also need to specify ZQM vs MQTT, or are you defaulting to ZMQ?

Reply:
Currently